package.json

```diff
@@ -112,7 +112,9 @@
     "nock": "^13.0.11",
     "nyc": "^15.1.0",
     "prettier": "^2.2.1",
-    "wait-for-expect": "^3.0.2",
+    "wait-for-expect": "^3.0.2"
+  },
+  "peerDependencies": {
     "web3": "^1.2.6"
```

Comment:
it should still be present on the `devDependencies` if is being used directly in the code. as `peerDependencies` is only to provide information when adding this project as dependency to another that is needed that library

Reply:
Yup just saw that 🤦 I added back the devDependency

We need to figure out which version we want to add to peerDependencies though, web3.js is currently in ^1.5 so I find it a bit hard to limit the peerDependencies to 1.2.6, I put ^1.x for now but I would need to dig a bit